webpack.config.js

```diff
@@ -61,7 +61,12 @@ const blockEntryPoints = () => {
 		{ absolute: true }
 	);
 
-	if ( ! coreBlockFiles.length ) {
+	const coreBlockAdminFiles = glob(
+		`${ pkg.config.coreThemeBlocksDir }/**/admin.js`,
+		{ absolute: true }
+	);
```

Comment:
We needed another entry point for the new `admin.js` files being added for core blocks, this seemed like a good place to add this.

Reply:
Thanks for this Geoff, I'm curious why this would be necessary though? Would we be applying separate functionality against a core block in the admin versus the FE?

Reply:
Are you talking about just using the `index.js` file that exists in the core block folders already and enqueuing those files instead? That feels wrong to me for some reason. I feel like having a separate `admin.js` file in the core block folders that only enqueues on the admin feels more specific and intentional on what it's used for, rather than a file that imports style files. I'm not locked to it though if you feel like it's totally unnecessary.

Reply:
I guess I'm more asking, JS functionality we would apply to core blocks at all. If we needed to write JS for a core block, I'd rather see us create an entirely custom block.

The reason we have an `index.js` file in the core block directories at all is to make sure that the webpack wp-scripts routine catches them as directories that need to be processed for setting up the `index.asset.php` file and compiling the imported css partials. Ideally, we would just have a `style.pcss` and/or an `editor.pcss` partial in the directory to kickoff processing, but webpack is looking for the index.js entry point to handle it's processing.

Reply:
@dpellenwood to answer one of your earlier questions- "Would we be applying separate functionality against a core block in the admin versus the FE?" Yes, we'd want to enqueue admin only JS for core blocks to filter block alignment support and/or disable block styles (i.e. quote default & plain styles).

Per your request [here](https://github.com/moderntribe/moose/pull/48#discussion_r1200673539) we moved the related core block admin JS to the individual block directories.